clang/lib/Lex/Pragma.cpp

```diff
@@ -421,7 +421,6 @@ void Preprocessor::HandlePragmaMark(Token &MarkTok) {
   SmallString<64> Buffer;
   CurLexer->ReadToEndOfLine(&Buffer);
   if (CurLexer){
-    CurLexer->ReadToEndOfLine(&Buffer);
```

Comment:
I think you removed the wrong one — you should keep the one inside `if (CurLexer)` not the one that precedes it I think.